lib/ansible/plugins/callback/logstash.py

```diff
@@ -11,7 +12,8 @@
     short_description: Sends events to Logstash
     description:
       - This callback will report facts and task events to Logstash https://www.elastic.co/products/logstash
-    version_added: "2.3"
+      - Before 2.10 only environment variables were available for configuring this plugin.
+    version_added: "2.10"
```

Comment:
The `version_added: "2.3"` was right before (this general plugin was adding in 2.3). I think @bcoca may have meant on the new ini way of providing the option (but I don't think that subkey is displayed by our docs build yet):
```
      server:
        description: Address of the Logstash server
        env:
          - name: LOGSTASH_SERVER
        default: localhost
        ini:
          - section: callback_logstash
            key: server
            version_added: "2.10"
```
You can document the versions options were added and it shows up in the docs, but it doesn't clarify which version works with the `ini` setting like the example above does:
```
      server:
        description: Address of the Logstash server
        version_added: "2.3"
        ...
```

Reply:
So I should do it like?
```
        ini:
          - section: callback_logstash
            key: server
            version_added: "2.10"
```